src/DotNetProjectFile.Analyzers/MsBuild/ProjectFileTypes.cs

```diff
@@ -4,12 +4,18 @@ public static class ProjectFileTypes
 {
     public static readonly IReadOnlyCollection<ProjectFileType> ProjectFile = [ProjectFileType.ProjectFile];
 
-    public static readonly IReadOnlyCollection<ProjectFileType> ProjectFileAndProps =
+    public static readonly IReadOnlyCollection<ProjectFileType> ProjectFile_Props =
```

Comment:
`Directory.Build.props` and `Directory.Packages.props` are also very much just `.props` file that can be used for defining anything, so shouldn't we include them here as well?

Reply:
@CptWesley For those rules (the default) `ProjectFileTYpes.All` exits. For some cases, those special cases are excluded. (The requirement to be an additional file for instance).